oshi-core/src/main/java/oshi/hardware/common/AbstractHardwareAbstractionLayer.java

```diff
@@ -37,12 +37,99 @@
 
     private static final long serialVersionUID = 1L;
 
-    protected ComputerSystem computerSystem;
+    private volatile ComputerSystem computerSystem;
 
-    protected CentralProcessor processor;
+    private volatile CentralProcessor processor;
 
-    protected GlobalMemory memory;
+    private volatile GlobalMemory memory;
 
-    protected Sensors sensors;
+    private volatile Sensors sensors;
 
+    /** {@inheritDoc} */
```

Comment:
All javadocs are inherited by default, so it is not necessary to specify the `@inheritDoc` tag unless we want to extend the javadoc. The below snippet explains some usages of the tag:
```java
interface A {
    /**
     * [A x]
     */
    void x();

    /**
     * [A y]
     * 
     * @param foo [A foo]
     * @param bar [A bar]
     * @throws IllegalArgumentException [A IllegalArgumentException]
     * @throws NullPointerException [A NullPointerException]
     */
    void y(int foo, String bar) throws IllegalArgumentException, NullPointerException;
}

interface B extends A {
    //inherits javadoc from A as is
    void x();

    /**
     * {@inheritDoc}
     * <p>
     * [B y]
     *
     * @param bar {@inheritDoc}[B bar]
     * @throws IllegalArgumentException [B IllegalArgumentException]{@inheritDoc}
     * @throws NullPointerException [B NullPointerException]
     */
    //extends method description of A
    //inherits foo param description from A as is
    //extends bar param description of A
    //extends IllegalArgumentException description of A
    //overrides NullPointerException description of A
    void y(int foo, String bar) throws IllegalArgumentException, NullPointerException;
}
```

Reply:
The vast majority of the "InheritDoc" statements were written by the maven javadoc plugin... I guess I can remove them but I don't see them as harmful and they serve as a visual reference to me... then again, so does the `@Override` annotation.